common/source/docs/common-fettec-onewire.rst

```diff
@@ -26,15 +31,16 @@ Connecting
 ==========
 
 The protocol supports up-to 25 ESCs, but the ArduPilot implementation limits this to 12.
-You can connect all ESCs to a single serial port using only a single signal wire from the autopilot's UART TX pin to the ESC's TLM pin and one GND wire.
+You must connect all ESCs to a single serial port using a single signal wire from the autopilot's UART TX pin to the ESC's TLM pin and one GND wire.
 If you are using 4in1 ESCs you can connect up-to 3 of them in this fashion.
 
 TODO: @teezeefpv add a figure here.
 
 FullDuplex with Diode
 =====================
 
-If halfduplex is not available or working, there is a workaround with a diode using the normal full duplex. This is also works with other "non-halfduplex" controllers like ESP32.
+If halfduplex is not available or working, there is a workaround with a diode using the normal full duplex.
```

Comment:
i fyou are trying for a line break, this does not work..must have a blank line,else it will be concatenated onto the previous line....you can force single space by using sphinx block markers though

Reply:
No, I was just making it neater